integration_tests/src/test/kotlin/utils/CommandHelper.kt

```diff
@@ -3,11 +3,33 @@ package utils
 import java.io.File
 import java.util.concurrent.TimeUnit
 
+
 fun String.runCommand(workingDir: File): ProcessResult = ProcessBuilder(*split("\\s".toRegex()).toTypedArray())
-    .directory(workingDir)
-    .redirectOutput(ProcessBuilder.Redirect.PIPE)
-    .redirectError(ProcessBuilder.Redirect.PIPE)
-    .start().run {
-        waitFor(60, TimeUnit.MINUTES)
-        ProcessResult(exitValue(), inputStream.bufferedReader().readText() + this.errorStream.bufferedReader().readText())
+    .directory(workingDir).run {
+        redirectOutputForCompatibleOS().start().run {
+            waitFor(processTimeout, TimeUnit.MINUTES)
+            ProcessResult(exitValue(), getOsSpecificOutput())
+        }
```

Comment:
how about 
```kotlin
    redirectOutputForCompatibleOS().startProcess()
    
    private fun ProcessBuilder.startProcess() = start().run {
            waitFor(processTimeout, TimeUnit.MINUTES)
            ProcessResult(exitValue(), getOsSpecificOutput())
        }
```
to avoid nested run?

Reply:
done :)